src/components/Helper.js

```diff
@@ -0,0 +1,20 @@
+/* eslint no-eval: 0 */
+const getRandomOperator = () => {
+  const operators = ['+', '-', '*', '/'];
+  const opIndex = Math.floor(Math.random() * 4);
+  return operators[opIndex];
+};
+const getRandomNumber = () => Math.floor(Math.random() * 20);
```

Comment:
pass 20 as an argument to this function. Keep it as a default argument

Reply:
Done